plugins/source/shopify/internal/shopify/shopify.go

```diff
@@ -39,7 +40,7 @@ type HTTPDoer interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
-func New(log zerolog.Logger, hc HTTPDoer, apiKey, apiSecret, accessToken, shopURL string, maxRetries int64) (*Client, error) {
+func New(log zerolog.Logger, hc HTTPDoer, apiKey, apiSecret, accessToken, shopURL string, maxRetries, pageSize int64) (*Client, error) {
```

Comment:
Nit: feels like this probably has enough arguments now that it should rather be turned into an options struct, which will help avoid passing the arguments in the wrong order.

Reply:
Fixed, not pretty but better.